src/components/behandlerdialog/skrivtilbehandler/SkrivTilBehandlerSkjema.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { SkjemaFeiloppsummering } from "@/components/SkjemaFeiloppsummering";
+import { FlexRow } from "@/components/Layout";
+import { Button } from "@navikt/ds-react";
+import { Form } from "react-final-form";
+import { VelgBehandler } from "@/components/behandlerdialog/skrivtilbehandler/VelgBehandler";
+import { BehandlerDTO } from "@/data/behandler/BehandlerDTO";
+import styled from "styled-components";
+import { MeldingTekstfelt } from "@/components/behandlerdialog/skrivtilbehandler/MeldingTekstfelt";
+import {
+  behandlerRefValidationErrors,
+  validerSkjemaTekster,
+} from "@/utils/valideringUtils";
+import { useFeilUtbedret } from "@/hooks/useFeilUtbedret";
+
+const texts = {
+  knappTekst: "Send til behandler",
+};
+
+const validationTexts = {
+  missingMeldingTekst: "Innholdet i meldingen er tomt", // TODO: se nærmere på teksten
+};
+
+const SkrivTilBehandlerFormWrapper = styled.div`
+  margin-top: 1.5em;
+`;
+
+const StyledForm = styled.form`
+  > * {
+    &:not(:last-child) {
+      margin-bottom: 1.5em;
+    }
+  }
+`;
+
+interface SkrivTilBehandlerSkjemaValues {
+  behandlerRef: string;
+  meldingTekst: string;
+}
+
+interface SkrivTilBehandlerSkjemaTekster {
+  meldingTekst: string;
+}
+
+type SkrivTilBehandlerSkjemaFeil = Partial<
+  Pick<SkrivTilBehandlerSkjemaValues, "behandlerRef" | "meldingTekst">
+>;
+
+export const MAX_LENGTH_BEHANDLER_MELDING = 2000; // TODO: må bli enige om noe her
+
+export const SkrivTilBehandlerSkjema = () => {
+  const [selectedBehandler, setSelectedBehandler] = useState<BehandlerDTO>();
+  const { harIkkeUtbedretFeil, resetFeilUtbedret, updateFeilUtbedret } =
+    useFeilUtbedret();
+
+  const validate = (
+    values: Partial<SkrivTilBehandlerSkjemaValues>
+  ): SkrivTilBehandlerSkjemaFeil => {
+    const friteksterFeil = validerSkjemaTekster<SkrivTilBehandlerSkjemaTekster>(
+      {
+        meldingTekst: {
+          maxLength: MAX_LENGTH_BEHANDLER_MELDING,
+          value: values.meldingTekst || "",
+          missingRequiredMessage: validationTexts.missingMeldingTekst,
+        },
+      }
+    );
+    const feilmeldinger: SkrivTilBehandlerSkjemaFeil = {
+      behandlerRef: behandlerRefValidationErrors(values.behandlerRef),
+      ...friteksterFeil,
+    };
+
+    updateFeilUtbedret(feilmeldinger);
+    return feilmeldinger;
+  };
+
+  const submit = (values: SkrivTilBehandlerSkjemaValues) => {
+    return; // TODO: POST-kall til API
+  };
+
+  return (
+    <SkrivTilBehandlerFormWrapper>
+      <Form onSubmit={submit} validate={validate}>
+        {({ handleSubmit, submitFailed, errors }) => (
+          <StyledForm onSubmit={handleSubmit}>
+            <VelgBehandler
+              selectedBehandler={selectedBehandler}
+              setSelectedBehandler={
+                setSelectedBehandler
+              } /* TODO: Skrive oss bort fra state her, bruke values fra form i stedet*/
+            />
```

Comment:
Behandlersøket har litt rar logikk, så vi har nå en state som kun brukes for å "tilfredsstille" APIet til søk-komponenten. Men tenker at utbedring av den kan komme senere 👍🏼